official/resnet/ctl/ctl_imagenet_test.py

```diff
@@ -83,7 +82,6 @@ def test_end_to_end_1_gpu(self):
       self.skipTest(
```

Comment:
we are probably missing this in other tests too, but instad of skipping, i think we can just run the same test with 1 CPU instead? (pass num_gpus=0). that way we get some coverage. 

also, after checking this in, remember to add internal BUILD files so that they run internally as well.

Reply:
Done.
Will follow up on this PR and add internal BUILD files so that we run this internally.